normandy/control/templates/control/edit_recipe.html

```diff
@@ -0,0 +1,14 @@
+{% extends "control/admin/base.html" %}
+
+{% block breadcrumbs %}
+  <div class="breadcrumbs">
+    <a href="/control">Recipes</a> <span class="separator">//</span> {{ recipe.name }}
```

Comment:
Not a big deal since this will change anyway, but you'll want to use the `url` template tag to generate this URL.